src/Lib.hs

```diff
@@ -116,8 +115,7 @@ doMakeCase b (RegularCase Function clauses ip) = do
   ins <- getIndent b (zeroIndex (p_line (iStart int)))
   replaceInterval b int
     $ T.unlines
-    $ fmap (T.replicate ins " " <>)
-    $ fmap replaceQuestion clauses
+    $ fmap ((T.replicate ins " " <>) . replaceQuestion) clauses
```

Comment:
Please revert this to the two `fmap`s

Reply:
ah, I was wondering what you were saying about `fmap (.)` but yes, this isn't a great refactor